Library/Homebrew/compat/macos.rb

```diff
@@ -80,7 +80,6 @@ def gcc_42_build_version
     alias_method :gcc_build_version, :gcc_42_build_version
 
     def llvm_build_version
-      DevelopmentTools.llvm_build_version
```

Comment:
What's the reason for retaining the method, but making it return `nil` instead? Could that be made more explicit by making the code look as follows? (I think an empty method is somewhat confusing.)

```ruby
# Brief explanation.
nil
```